packages/builder/src/components/backend/DataTable/buttons/grid/GridColumnsSettingButton.svelte

```diff
@@ -48,7 +48,7 @@
   </svelte:fragment>
   <ColumnsSettingContent
     columns={$tableColumns}
-    canSetRelationshipSchemas={isEnabled(FeatureFlag.ENRICHED_RELATIONSHIPS)}
+    canSetRelationshipSchemas
```

Comment:
Do we not want to remove this property altogether?

Reply:
This is used internally within a self. I will check if it can be cleanup and rely only on the backend data

Reply:
Worked fine :)